content/en/docs/spinnaker-user-guides/pacrd/_index.md

```diff
@@ -463,6 +463,36 @@ Events:
   Warning  PipelineValidationFailed  0s (x4 over 3s)        pipelines  artifact with id "a-nonsense-value" and name "" could not be found for this pipeline
 ```
 
+## Setting up New Relic
+
+If you want to setup a New Relic configuration you can add your license to your `patch.yaml` file like:
+
+   ```yaml
+   # file: patch.yaml
+   apiVersion: v1
+   kind: ConfigMap
+   metadata:
+     name: pacrd-config
+     namespace: spinnaker
+   data:
+     pacrd.yaml: |
+       spinnakerServices:
+         # NOTE: change `spinnaker` to your namespace name here
+         front50: https://spin-front50.spinnaker:8080
+         orca: https://spin-orca.spinnaker:8083
+       # fiatServiceAccount: <fiatServiceAccount>
+       newRelicLicense: <newRelicLicense>
+   ```
+
+Since reconciliation happens multiple times per minute, metrics are send only on minutes 0-3 each hour. Also error messages have obfuscated urls and application names.
```

Comment:
```suggestion
Since reconciliation happens multiple times per minute, Armory sends metrics only during the first three minutes of each hour.   Error messages contain obfuscated URLs, application names, and pipeline names.
```